project/Dependencies.scala

```diff
@@ -74,7 +75,7 @@ object Dependencies {
     // ssl-config
     val sslConfigCore = Def.setting { "com.typesafe" %% "ssl-config-core" % sslConfigVersion } // ApacheV2
 
-    val lmdb = "org.lmdbjava" % "lmdbjava" % "0.6.1" // ApacheV2, OpenLDAP Public License
+    val lmdb = "org.lmdbjava" % "lmdbjava" % "0.7.0" // ApacheV2, OpenLDAP Public License
```

Comment:
(important to doublecheck)

Reply:
change log here https://github.com/lmdbjava/lmdbjava/wiki/Change-Log
should be fine, if it couldn't read old data it would be a dead product